lua/conform/formatters/deno_fmt.lua

```diff
@@ -16,11 +34,29 @@ return {
   },
   command = "deno",
   args = function(self, ctx)
-    return {
+    local extension = extensions[vim.bo[ctx.buf].filetype]
+    local formatter_args = {
       "fmt",
       "-",
       "--ext",
-      extensions[vim.bo[ctx.buf].filetype],
+      extension,
     }
+
+    if
+      vim.tbl_get(unstable_extensions, extension)
```

Comment:
No need to complicate things, we can just do
```suggestion
      unstable_extensions[extension]
```

Reply:
True, I was overthinking it 👍